Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -12,157 +11,97 @@ To get started with the Mixed Reality Toolkit you will need:
-
-![](../External/ReadMeImages/MixedRealityToolkitConfigurationProfileScreens/MRTK_ActiveConfiguration.png)
-
-Clicking on this profile will show the configuration screens for the Mixed Reality Toolkit:
-
-![](../External/ReadMeImages/MixedRealityToolkitConfigurationProfileScreens/MRTK_MixedRealityToolkitConfigurationScreen.png)
-
-From here you can navigate to all the configuration profiles for the MRTK, including:
-
-> The "Default" profiles provided by the Mixed Reality Toolkit are locked by default, so when you view these in the inspector they will appear greyed out.  This is to ensure you always have a common default for any project.  We recommend you create your own profiles (see below) when you need to customize the configuration for your project.
-
-* Main Mixed Reality Toolkit Configuration
-* Camera Settings
-* Input System Settings
-* Boundary Visualization Settings
-* Teleporting Settings
-* Spatial Awareness Settings
-* Diagnostics Settings
-* Additional Services Settings
-* Input Actions Settings
-* Input Actions Rules
-* Pointer Configuration
-* Gestures Configuration
-* Speech Commands
-* Controller Mapping Configuration
-* Controller Visualization Settings
-
-As you can see there are lots of options available and more will come available as we progress through the beta.
-
-When you start a new project, we provide a default set of configurations with every option turned on, styled for a fully cross platform project.  These defaults are "Locked" to ensure you always have a common start point for your project and we encourage you to start defining your own settings as your project evolves.  For this we provide options to either:
-
-* Copy the defaults in to a new profile for you to start customizing it for your project
-* Start afresh with a brand-new profile.
-
-![](../External/ReadMeImages/MRTK_CopyCreateConfigurationOptions.png)
-
-When profiles are created by the MRTK, they are then placed in the following folder:
-
-> "Assets\MixedRealityToolkit-Generated\CustomProfiles"
-
-At each step in the configuration, you can choose to remove and create a new profile, or simply copy the existing settings and continue to customize:
-
-![](../External/ReadMeImages/MRTK_CopyProfileOptions.png)
-
-### **[For more information on customizing the Configuration Profiles](MixedRealityConfigurationGuide.md)**
-Please check out the [Mixed Reality Configuration Guide](MixedRealityConfigurationGuide.md)
+5. Hit play
+Press the play button to start the scene. You are now ready to start building your project, just start adding content and get building. Here are suggested next steps.
 
 ## Get building your project
 
-Now your project is up and running, you can start building your Mixed Reality project.  
-
-For more information on the rest of the toolkit, please check the following guides:
+Now your project is up and running, you can start building your Mixed Reality project. Here are some suggested next steps:
```

Comment:
> Here are some suggested next steps: [](start = 86, length = 36)

Also, these steps don't appear to have anything to do with "building my project", maybe we can put them in another place? So that if a user scans for and stops at "Get building your project" we don't distract him from the task of building it

Reply:
Updated to say 'next steps'

---
In reply to: [273693048](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3904#discussion_r273693048) [](ancestors = 273693048)